{"nodes":[{"pos":[522,613],"content":"Represents the exception that is thrown when errors are generated using ADO.NET components.","needQuote":true,"needEscape":true,"nodes":[{"content":"Represents the exception that is thrown when errors are generated using ADO.NET components.","pos":[0,91]}]},{"pos":[1129,1249],"content":"Initializes a new instance of the <xref href=\"System.Data.DataException\"></xref> class. This is the default constructor.","needQuote":true,"needEscape":true,"nodes":[{"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.Data.DataException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class.","pos":[0,87],"source":"Initializes a new instance of the <xref href=\"System.Data.DataException\"></xref> class."},{"content":"This is the default constructor.","pos":[88,120]}]},{"pos":[1769,1882],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.Data.DataException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class with the specified string.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.Data.DataException\"></xref> class with the specified string."},{"pos":[2012,2063],"content":"The string to display when the exception is thrown.","needQuote":true,"needEscape":true,"nodes":[{"content":"The string to display when the exception is thrown.","pos":[0,51]}]},{"pos":[2758,2902],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.Data.DataException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class with the specified serialization information and context.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.Data.DataException\"></xref> class with the specified serialization information and context."},{"pos":[3169,3226],"content":"The data necessary to serialize or deserialize an object.","needQuote":true,"needEscape":true,"nodes":[{"content":"The data necessary to serialize or deserialize an object.","pos":[0,57]}]},{"pos":[3322,3399],"content":"Description of the source and destination of the specified serialized stream.","needQuote":true,"needEscape":true,"nodes":[{"content":"Description of the source and destination of the specified serialized stream.","pos":[0,77]}]},{"pos":[3916,4049],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.Data.DataException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class with the specified string and inner exception.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.Data.DataException\"></xref> class with the specified string and inner exception."},{"pos":[4061,4315],"content":"You can create a new exception that catches an earlier exception. The code that handles the second exception can make use of the additional information from the earlier exception, also called an inner exception, to examine the cause of the initial error.","needQuote":true,"needEscape":true,"extradata":"MT","nodes":[{"content":"You can create a new exception that catches an earlier exception. The code that handles the second exception can make use of the additional information from the earlier exception, also called an inner exception, to examine the cause of the initial error.","pos":[0,254],"nodes":[{"content":"You can create a new exception that catches an earlier exception.","pos":[0,65]},{"content":"The code that handles the second exception can make use of the additional information from the earlier exception, also called an inner exception, to examine the cause of the initial error.","pos":[66,254]}]}]},{"pos":[4471,4522],"content":"The string to display when the exception is thrown.","needQuote":true,"needEscape":true,"nodes":[{"content":"The string to display when the exception is thrown.","pos":[0,51]}]},{"pos":[4596,4630],"content":"A reference to an inner exception.","needQuote":true,"needEscape":true,"nodes":[{"content":"A reference to an inner exception.","pos":[0,34]}]}],"content":"### YamlMime:ManagedReference\nitems:\n- uid: System.Data.DataException\n  id: DataException\n  children:\n  - System.Data.DataException.#ctor\n  - System.Data.DataException.#ctor(System.String)\n  - System.Data.DataException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  - System.Data.DataException.#ctor(System.String,System.Exception)\n  langs:\n  - csharp\n  name: DataException\n  nameWithType: DataException\n  fullName: System.Data.DataException\n  type: Class\n  summary: Represents the exception that is thrown when errors are generated using ADO.NET components.\n  syntax:\n    content: 'public class DataException : SystemException'\n  inheritance:\n  - System.Exception\n  - System.SystemException\n  implements: []\n  inheritedMembers: []\n  platform:\n  - net462\n- uid: System.Data.DataException.#ctor\n  id: '#ctor'\n  parent: System.Data.DataException\n  langs:\n  - csharp\n  name: DataException()\n  nameWithType: DataException.DataException()\n  fullName: System.Data.DataException.DataException()\n  type: Constructor\n  assemblies:\n  - System.Data\n  namespace: System.Data\n  summary: Initializes a new instance of the <xref href=\"System.Data.DataException\"></xref> class. This is the default constructor.\n  syntax:\n    content: public DataException ();\n    parameters: []\n  overload: System.Data.DataException.#ctor*\n  exceptions: []\n  platform:\n  - net462\n- uid: System.Data.DataException.#ctor(System.String)\n  id: '#ctor(System.String)'\n  parent: System.Data.DataException\n  langs:\n  - csharp\n  name: DataException(String)\n  nameWithType: DataException.DataException(String)\n  fullName: System.Data.DataException.DataException(String)\n  type: Constructor\n  assemblies:\n  - System.Data\n  namespace: System.Data\n  summary: Initializes a new instance of the <xref href=\"System.Data.DataException\"></xref> class with the specified string.\n  syntax:\n    content: public DataException (string s);\n    parameters:\n    - id: s\n      type: System.String\n      description: The string to display when the exception is thrown.\n  overload: System.Data.DataException.#ctor*\n  exceptions: []\n  platform:\n  - net462\n- uid: System.Data.DataException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'\n  parent: System.Data.DataException\n  langs:\n  - csharp\n  name: DataException(SerializationInfo,StreamingContext)\n  nameWithType: DataException.DataException(SerializationInfo,StreamingContext)\n  fullName: System.Data.DataException.DataException(SerializationInfo,StreamingContext)\n  type: Constructor\n  assemblies:\n  - System.Data\n  namespace: System.Data\n  summary: Initializes a new instance of the <xref href=\"System.Data.DataException\"></xref> class with the specified serialization information and context.\n  syntax:\n    content: protected DataException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);\n    parameters:\n    - id: info\n      type: System.Runtime.Serialization.SerializationInfo\n      description: The data necessary to serialize or deserialize an object.\n    - id: context\n      type: System.Runtime.Serialization.StreamingContext\n      description: Description of the source and destination of the specified serialized stream.\n  overload: System.Data.DataException.#ctor*\n  exceptions: []\n  platform:\n  - net462\n- uid: System.Data.DataException.#ctor(System.String,System.Exception)\n  id: '#ctor(System.String,System.Exception)'\n  parent: System.Data.DataException\n  langs:\n  - csharp\n  name: DataException(String,Exception)\n  nameWithType: DataException.DataException(String,Exception)\n  fullName: System.Data.DataException.DataException(String,Exception)\n  type: Constructor\n  assemblies:\n  - System.Data\n  namespace: System.Data\n  summary: Initializes a new instance of the <xref href=\"System.Data.DataException\"></xref> class with the specified string and inner exception.\n  remarks: You can create a new exception that catches an earlier exception. The code that handles the second exception can make use of the additional information from the earlier exception, also called an inner exception, to examine the cause of the initial error.\n  syntax:\n    content: public DataException (string s, Exception innerException);\n    parameters:\n    - id: s\n      type: System.String\n      description: The string to display when the exception is thrown.\n    - id: innerException\n      type: System.Exception\n      description: A reference to an inner exception.\n  overload: System.Data.DataException.#ctor*\n  exceptions: []\n  platform:\n  - net462\nreferences:\n- uid: System.SystemException\n  isExternal: false\n  name: System.SystemException\n- uid: System.Data.DataException.#ctor\n  parent: System.Data.DataException\n  isExternal: false\n  name: DataException()\n  nameWithType: DataException.DataException()\n  fullName: System.Data.DataException.DataException()\n- uid: System.Data.DataException.#ctor(System.String)\n  parent: System.Data.DataException\n  isExternal: false\n  name: DataException(String)\n  nameWithType: DataException.DataException(String)\n  fullName: System.Data.DataException.DataException(String)\n- uid: System.String\n  parent: System\n  isExternal: true\n  name: String\n  nameWithType: String\n  fullName: System.String\n- uid: System.Data.DataException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  parent: System.Data.DataException\n  isExternal: false\n  name: DataException(SerializationInfo,StreamingContext)\n  nameWithType: DataException.DataException(SerializationInfo,StreamingContext)\n  fullName: System.Data.DataException.DataException(SerializationInfo,StreamingContext)\n- uid: System.Runtime.Serialization.SerializationInfo\n  parent: System.Runtime.Serialization\n  isExternal: false\n  name: SerializationInfo\n  nameWithType: SerializationInfo\n  fullName: System.Runtime.Serialization.SerializationInfo\n- uid: System.Runtime.Serialization.StreamingContext\n  parent: System.Runtime.Serialization\n  isExternal: true\n  name: StreamingContext\n  nameWithType: StreamingContext\n  fullName: System.Runtime.Serialization.StreamingContext\n- uid: System.Data.DataException.#ctor(System.String,System.Exception)\n  parent: System.Data.DataException\n  isExternal: false\n  name: DataException(String,Exception)\n  nameWithType: DataException.DataException(String,Exception)\n  fullName: System.Data.DataException.DataException(String,Exception)\n- uid: System.Exception\n  parent: System\n  isExternal: true\n  name: Exception\n  nameWithType: Exception\n  fullName: System.Exception\n- uid: System.Data.DataException.#ctor*\n  parent: System.Data.DataException\n  isExternal: false\n  name: DataException\n  nameWithType: DataException.DataException\n"}